app/src/main/res/layout/content_tourbook.xml

```diff
@@ -69,20 +68,18 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_weight="1"
-            android:background="#ffff0000"
             android:buttonTint="@android:color/black"
             android:onClick="showBotches"
-            android:text="Säcke" />
+            android:text="Säcke \u26A1"/>
```

Comment:
Seems not so easy. However you can add at least a separate resource string definition e.g. `<string name="botch_text_symbol">Säcke \u26A1</string>` to avoid hard coded strings in the actual source code.

Reply:
No, I'm fine with hard-coding. My intension was to have the unicode symbol defined in only one place (for the sake of maybe changing it in the future) and not twice.

Reply:
But with the proposed approach it would be the case. It is defined twice but only in one file and very close together.

Reply:
True, ok